operations/rates.md

```diff
@@ -49,222 +52,159 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the Enterprises. If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `Extent` | [Rate extent](rates.md#rate-extent) | required | Extent of data to be returned. |
+| `RateIds` | array of string | optional, max 1000 items | Unique identifiers of the requested [Rates](rates.md#rate). |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months |  |
```

Comment:
Missing description

Reply:
fixed as a part of this [commit](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/aea81f3fe83c61522fc3bcfd401a5535459b7ab0)